packages/kbn-unified-field-list/src/components/field_stats/field_stats.tsx

```diff
@@ -336,6 +347,22 @@ const FieldStatsComponent: React.FC<FieldStatsProps> = ({
       : messageNoAnalysis;
   }
 
+  const unsupportedReasonForQuery = getReasonIfFieldStatsUnavailableForQuery(
+    query,
+    FIELD_DATA_LABEL
+  );
+  if (unsupportedReasonForQuery) {
+    const messageUnsupportedReason = (
+      <EuiCallOut size="s" iconType="warning" color="warning" title={unsupportedReasonForQuery} />
+    );
+
+    return overrideMissingContent
+      ? overrideMissingContent({
+          reason: 'unsupported',
+          element: messageUnsupportedReason,
+        })
+      : messageUnsupportedReason;
+  }
```

Comment:
QQ: should we also skip the request to ES in this case?

Reply:
Yes! Definitely. Thanks for catching that.

Reply:
Thanks!